CSS/about.css

```diff
@@ -0,0 +1,161 @@
+#headline-section {
+  background: url("../Images/waraAboutMobile.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.headline-title-text p {
+  text-align: center;
+}
+
+.headline-title-text h2 {
+  text-align: center;
+}
+
+.headline-box-text {
+  background: #ffffffbf;
+  border: 2px solid #2a282829;
+}
+
+.headline-box-text p {
+  text-align: center;
```

Comment:
Let's avoid code repetition and we could work with styling multiple classes at once 😉 

For example:
```
.headline-title-text h2,
.headline-title-text p,
.headline-box-text p {
  text-align: center;
}
```